deftree/build_deftree.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/TuneLab/go-truss/deftree/svcparse"
-	"github.com/TuneLab/go-truss/truss/protostage"
+	"github.com/TuneLab/go-truss/truss/protostuff"
```

Comment:
💩 smell. Surely there is a more descriptive name than `stuff`. Perhaps the package needs to be broken apart?